egapro/views.py

```diff
@@ -189,6 +189,15 @@ def is_declaration(self, response, data):
         return True
 
 
+@app.route("/index/{siren}")
```

Comment:
Should we restrict access to this route ?

Reply:
Yes we should. I didn't have the ambition to implement such a feature in this same MR. Should I?